app/graph/types/episode_enum.rb

```diff
@@ -1,7 +1,7 @@
 # enum Episode { NEWHOPE, EMPIRE, JEDI }
 module Types
-  EpisodeEnum = GraphQL::EnumType.define do
-    name "Episode"
+  class EpisodeEnum < BaseEnum
+    graphql_name "Episode"
```

Comment:
This is a missing feature in the graphql-ruby gem. I asked @esjee to see if he can create a PR so we don't have to specify the graphql_name if we can infer the name from the class, something that can be done on types.